app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] Kindly, make sure that you create a column author with `author_id` as the foreign key so that your user model could look as below:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```

Reply:
Hello @RolandM99 
My tables have a columns `user_id` which references the `User` table.
I think because of the naming convention of rails I don't really need to add the `foreign_key: 'author_id'` since its `user_id`